packages/runtime/client/rest/src/index.ts

```diff
@@ -142,8 +142,13 @@ export interface RestAPI {
     txId: TxId
   ): Promise<TransactionDetails>;
   //   submitTransaction: Transaction.PUT; // - Jamie is it this one? https://docs.marlowe.iohk.io/api/create-transaction-by-id? If so, lets unify
-
-  //   getWithdrawals: Withdrawals.GET; // - https://docs.marlowe.iohk.io/api/get-withdrawals
+  /**
+   * Get published withdrawal transactions.
+   * @see {@link https://docs.marlowe.iohk.io/api/get-withdrawals}
+   */
+  getWithdrawals(
```

Comment:
Looking at the backend, this seems to be missing roleCurrency

Reply:
Nice catch. I've addressed it.